package-lock.json

```
@@ -24,7 +24,7 @@
                 "command-exists": "~1.2.9",
                 "compare-versions": "~3.6.0",
                 "compression": "~1.7.4",
-                "croner": "~6.0.5",
+                "croner": "^8.1.0",
```

You forgot to run `npm install` ^^

```suggestion
                "croner": "~8.1.0",
```